Makefile

```
@@ -132,8 +129,16 @@ release: all deps docs $(SMF_MANIFESTS)
 	(cd $(RELSTAGEDIR) && $(TAR) -I pigz -cf $(ROOT)/$(RELEASE_TARBALL) root site)
 	@rm -rf $(RELSTAGEDIR)
 
-regen_docs:
-	$(NODE) ./tools/gendocs.js docs/static.md lib/endpoints > docs/index.md
+./node_modules/.bin/doctoc:
+	npm install
```

```suggestion
	$(NPM) install
```

should also be the NPM var